lib/beaker/command_factory.rb

```diff
@@ -16,7 +16,10 @@ def assertions
     end
 
     def execute(command, options={}, &block)
-      result = self.exec(Command.new(command), options)
+      options[:prepend_cmds] ? 
```

Comment:
Let's do the prepend command processing in Command - so just pass the option through here.